app-service-certificate-standard/azuredeploy.json

```diff
@@ -0,0 +1,129 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "certificateOrderName":{
+         "type":"string",
+         "metadata":{
+            "description":"Name of the App Service Certificate"
+         }
+      },
+      "productType":{
+         "type":"string",
+         "allowedValues":[
+            "StandardDomainValidatedSsl",
+            "StandardDomainValidatedWildCardSsl"
+         ],
+         "defaultValue":"StandardDomainValidatedSsl",
+         "metadata":{
+            "description":"App Service Certificate type"
+         }
+      },
+      "keyVaultId":{
```

Comment:
you can probably ask the user for key vault resource group and name and build the resource id yourself using the resourceId function